packages/asset-swapper/test/dex_sampler_test.ts

```diff
@@ -326,6 +341,104 @@ describe('DexSampler tests', () => {
             );
             expect(quotes).to.deep.eq(expectedQuotes);
         });
+
+        describe('LiquidityProvider Operations', () => {
```

Comment:
I don't know how I feel about these tests. These seem be partially testing the test contracts. I'd also rather we didn't do on-chain tests in this package.

Do you want to unit test the `DexOrderSampler` ops? If so, extending the `MockSamplerContract` is the way to go, like how the other tests in this file do it. This would also remove the need for the new test contracts.

But if you want to unit test `ERC20BridgeSampler` integration with liquidity providers, that should be done in the `contracts/erc20-bridge-sampler` package.

Reply:
that's a wise point of view, I'm going to move my tests in `erc20-bridge-sampler` test package

Reply:
I refactored and moved those unit tests to `erc20-bridge-sampler`. I also added extra unit tests using `MockSamplerContract`